lib/grape/dsl/settings.rb

```diff
@@ -162,6 +162,18 @@ def within_namespace(&_block)
 
         result
       end
+
+      private
+
+      # Obtains the current class :inheritable_setting. If available, it returns the superclass's :inheritable_setting.
+      # Otherwise, a clean :inheritable_setting is returned.
+      def obtain_top_level_setting
```

Comment:
I don't love the `obtain` name, maybe just `get`, another more common variation is `build`?